packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts

```diff
@@ -412,23 +422,26 @@ export const createAzureCommunicationMeetingAdapter = async ({
   displayName,
   credential,
   endpoint,
-  chatThreadId,
-  callLocator
+  meetingArgs
 }: AzureCommunicationMeetingAdapterArgs): Promise<MeetingAdapter> => {
-  const callAdapter = await createAzureCommunicationCallAdapter({
+  const locator = 'meetingLink' in meetingArgs ? meetingArgs : meetingArgs.callLocator;
```

Comment:
This logic is used in at least two places, and even otherwise I'd request that we add a helper function for readability:

```ts
const isTeamsMeetingLinkLocator(locator: CallAndChatMeetingArgs | TeamsMeetingLinkLocator): locator is TeamsMeetingLinkLocator => {
  return 'meetingLink' in locator;
}

// Use as
const locator = isTeamsMeetingLinkLocator(meetingArgs) ? meetingArgs : meetingArgs.callLocator;

```

Reply:
updated, I'm pleased typescript was able to play well so well with this